beer-schema.sql

```diff
@@ -1,132 +1,127 @@
 );
 
-CREATE TABLE BeerStyle(
-    Style VARCHAR(255),
-    CONSTRAINT pk_BeerStyle_Style
-        PRIMARY KEY (Style)
+CREATE TABLE Brewers (
+  BrewerId INT,
+  CONSTRAINT pk_Brewers_BrewId
+    PRIMARY KEY (BrewerId)
 );
 
-CREATE TABLE Food(
-    FoodName VARCHAR(255),
-    Style VARCHAR(255),
-    CONSTRAINT pk_Food_FoodName
-        PRIMARY KEY (FoodName),
-    CONSTRAINT fk_Food_Style
-        FOREIGN KEY (Style)
-        REFERENCES BeerStyle(Style)
-        ON UPDATE CASCADE ON DELETE SET NULL
+CREATE TABLE BeerStyles (
+  Style VARCHAR(255),
+  CONSTRAINT pk_BeerStyles_Style
+    PRIMARY KEY (Style)
 );
 
-CREATE TABLE Brewer (
-    BrewerId INT,
-    CONSTRAINT pk_Brewer_BrewId
-        PRIMARY KEY (BrewerId)
+CREATE TABLE Beers (
+  BeerId INT,
+  BeerName VARCHAR(255),
+  ABV FLOAT,
+  BrewerId INT,
+  Style VARCHAR(255),
+  CONSTRAINT pk_Beers_BeerId
+    PRIMARY KEY (BeerId),
+  CONSTRAINT fk_Beers_BrewerId
+    FOREIGN KEY (BrewerId)
+    REFERENCES Brewers(BrewerId)
+    ON UPDATE CASCADE ON DELETE SET NULL,
+  CONSTRAINT fk_Beers_Style
+    FOREIGN KEY (Style)
+    REFERENCES BeerStyles(Style)
+    ON UPDATE CASCADE ON DELETE CASCADE
 );
 
-CREATE TABLE Beer (
-    BeerId INT NOT NULL,
-    BeerName VARCHAR(255) NOT NULL,
-    ABV FLOAT NOT NULL,
-    Style VARCHAR(255),
-    BrewerId INT,
-    CONSTRAINT pk_Beer_BeerId
-        PRIMARY KEY (BeerId),
-    CONSTRAINT fk_Beer_BrewerId
-        FOREIGN KEY (BrewerId)
-        REFERENCES Brewer(BrewerId)
-        ON UPDATE CASCADE ON DELETE SET NULL
+CREATE TABLE ViewHistory (
+  ViewId INT AUTO_INCREMENT,
+  Created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  BeerId INT,
+  UserName VARCHAR(255),
+  CONSTRAINT pk_ViewHistory_ViewId
+    PRIMARY KEY (ViewId),
+  CONSTRAINT fk_ViewHistory_UserName
+    FOREIGN KEY (UserName)
+    REFERENCES Persons(UserName)
+    ON UPDATE CASCADE ON DELETE SET NULL,
+  CONSTRAINT fk_ViewHistory_BeerId
+    FOREIGN KEY (BeerId)
+    REFERENCES Beers(BeerId)
+    ON UPDATE CASCADE ON DELETE SET NULL
 );
 
-CREATE TABLE ViewHistory (
-    ViewId INT AUTO_INCREMENT,
-    Created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    Username VARCHAR(255),
-    BeerId INT,
-    CONSTRAINT pk_ViewHistory_ViewId
-        PRIMARY KEY (ViewId),
-    CONSTRAINT fk_ViewHistory_Username
-        FOREIGN KEY (Username)
-        REFERENCES Person(Username)
-        ON UPDATE CASCADE ON DELETE SET NULL,
-    CONSTRAINT fk_ViewHistory_BeerId
-        FOREIGN KEY (BeerId)
-        REFERENCES Beer(BeerId)
-        ON UPDATE CASCADE ON DELETE SET NULL
+CREATE TABLE BeerReviews (
+	ReviewID INT AUTO_INCREMENT,
+	BeerID INT,
+	Appearance FLOAT,
+	Aroma FLOAT,
+	Palate FLOAT,
+	Taste FLOAT,
+	Overall FLOAT,
+	Time TIMESTAMP,
+	Text VARCHAR(1024),
+	UserName VARCHAR(255),
+	CONSTRAINT pk_BeerReviews_reviewID PRIMARY KEY (ReviewID),
+	CONSTRAINT fk_BeerReviews_beerID FOREIGN KEY (BeerID)
+		REFERENCES Beers(BeerID)
+		ON UPDATE CASCADE ON DELETE SET NULL
```

Comment:
Inconsistent indenting with the previous CREATE TABLE statements. Sorry I don't know how picky we want to be. My bad. You can ignore this if you want.

Reply:
fixed all